src/mumble/os_win.cpp

```diff
@@ -20,6 +20,7 @@ extern "C" {
 	void mumble_speex_init();
 };
 
+#undef PATH_MAX
```

Comment:
PATH_MAX is already overloaded in os_win.cpp in the sense that it's also used for the comment.

Let's just rename it instead.

    #define DUMP_BUFFER_SIZE 1024

and use that instead?